packages/zoe/src/contracts/priceAggregatorChainlink.js

```diff
@@ -557,122 +656,170 @@ const start = async (
    * @param {Timestamp} blockTimestamp
    */
   const eligibleForSpecificRound = (
-    _oracle,
-    _queriedRoundId,
+    oracleAddr,
+    queriedRoundId,
     blockTimestamp,
   ) => {
-    const error = validateOracleRound(_oracle, _queriedRoundId, blockTimestamp);
-    if (TimeMath.absValue(rounds.get(_queriedRoundId).startedAt) > 0n) {
-      return acceptingSubmissions(_queriedRoundId) && error.length === 0;
+    const error = validateOracleRound(
+      oracleAddr,
+      queriedRoundId,
+      blockTimestamp,
+    );
+    if (TimeMath.absValue(rounds.get(queriedRoundId).startedAt) > 0n) {
+      return acceptingSubmissions(queriedRoundId) && error === null;
     } else {
-      return delayed(_oracle, _queriedRoundId) && error.length === 0;
+      return delayed(oracleAddr, queriedRoundId) && error === null;
     }
   };
 
   /**
-   * @param {OracleKey} _oracle
+   * @param {string} oracleAddr
    */
-  const getStartingRound = _oracle => {
+  const getStartingRound = oracleAddr => {
     const currentRound = reportingRoundId;
     if (
       currentRound !== 0n &&
-      currentRound === oracleStatuses.get(_oracle).endingRound
+      currentRound === oracleStatuses.get(oracleAddr).endingRound
     ) {
       return currentRound;
     }
     return add(currentRound, 1);
   };
 
   /**
-   * @type {Omit<import('./priceAggregator').PriceAggregatorContract['creatorFacet'], 'makeOracleInvitation'> & {
-   *   getRoundData(_roundId: BigInt): Promise<any>,
-   *   oracleRoundState(_oracle: OracleKey, _queriedRoundId: BigInt): Promise<any>
+   * @type {Omit<import('./priceAggregator').PriceAggregatorContract['creatorFacet'], 'initOracle'> & {
+   *   initOracle: (instance) => Promise<OracleAdmin<PriceRound>>,
+   *   getRoundData(roundId: bigint | number): Promise<RoundData>,
+   *   oracleRoundState(oracleAddr: string, queriedRoundId: BigInt): Promise<any>
    * }}
    */
   const creatorFacet = Far('PriceAggregatorChainlinkCreatorFacet', {
-    deleteOracle: async oracleKey => {
-      const records = keyToRecords.get(oracleKey);
+    /**
+     * An "oracle invitation" is an invitation to be able to submit data to
+     * include in the priceAggregator's results.
+     *
+     * The offer result from this invitation is a OracleAdmin, which can be used
+     * directly to manage the price submissions as well as to terminate the
+     * relationship.
+     *
+     * @param {string} oracleAddr
+     */
+    makeOracleInvitation: async oracleAddr => {
+      /**
+       * If custom arguments are supplied to the `zoe.offer` call, they can
+       * indicate an OraclePriceSubmission notifier and a corresponding
+       * `shiftValueOut` that should be adapted as part of the priceAuthority's
+       * reported data.
+       *
+       * @param {ZCFSeat} seat
+       * @returns {Promise<{admin: OracleAdmin<PriceRound>, invitationMakers: {makePushPriceInvitation: (result: PriceRound) => Promise<Invitation<void>>} }>}
```

Comment:
```suggestion
       * @returns {Promise<{admin: OracleAdmin<PriceRound>, invitationMakers: {MakePushPriceInvitation: (result: PriceRound) => Promise<Invitation<void>>} }>}
```

The [sparsely-documented continuing invitation pattern](https://github.com/Agoric/documentation/issues/667) seems to including capitalizing the fields of invitation makers. For example:

```js
    invitationMakers: Far('invitation makers', {
      AdjustBalances: vault.makeAdjustBalancesInvitation,
      CloseVault: vault.makeCloseInvitation,
      TransferVault: vault.makeTransferInvitation,
    }),
```

I think that's to avoid colliding with existing property names, as noted for `assertKeywordName` in Zoe.

I can imagine this argument is not very strong, but if we're not going to follow the pattern in this respect, I would want to discuss it more widely.

Reply:
I guess the `AdjustBalances` example suggests just `PushPrice`.

Reply:
Ah... and I think the method name is supposed to match the invitation description, which is also `PushPrice`.

Reply:
Good call. `invitationMakers.makePushPriceInvitation` is redundant. Changed to `invitationMakers.PushPrice`.